opensuse-tomcat-jre11-juli-image/pom.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2017-2023 Micro Focus or one of its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.github.cafapi</groupId>
+        <artifactId>opensuse-tomcat-jre11-image-aggregator</artifactId>
+        <version>2.7.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>opensuse-tomcat-jre11-juli-image</artifactId>
+    <packaging>pom</packaging>
+    
+    <name>openSUSE Tomcat JRE11 Juli image</name>
+    <properties>
+        <maven.install.skip>true</maven.install.skip>
+        <DOCKER_HUB_PUBLIC>${dockerHubPublic}</DOCKER_HUB_PUBLIC>
+    </properties>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>io.fabric8</groupId>
+                <artifactId>docker-maven-plugin</artifactId>
+                <version>0.34.0</version>
+                <executions>
+                    <execution>
+                        <id>docker-build</id>
+                        <phase>compile</phase>
+                        <goals>
+                            <goal>build</goal>
+                        </goals>
+                    </execution>
+                    <execution>
+                        <id>docker-push</id>
+                        <phase>deploy</phase>
+                        <goals>
+                            <goal>push</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <images>
+                        <image>
+                            <name>${dockerCafImagePrefix}opensuse-tomcat-jre11-juli${dockerProjectVersion}</name>
+                            <build>
+                                <dockerFileDir>.</dockerFileDir>
+                                <assembly>
+                                    <basedir>/</basedir>
+                                </assembly>
```

Comment:
Did you have a purpose for this entry, it creates a warning in the build logs.

Reply:
I was following what was done for [JRE8 here](https://github.com/CAFapi/opensuse-tomcat-image/blob/develop/opensuse-tomcat-juli-image/pom.xml#L99), as explained earlier, there are no tests as part of this image currently, there are however in JRE8

Reply:
What was the warning log so can test/look out for it?

Reply:
> I was following what was done for [JRE8 here](https://github.com/CAFapi/opensuse-tomcat-image/blob/develop/opensuse-tomcat-juli-image/pom.xml#L99), as explained earlier, there are no tests as part of this image currently, there are however in JRE8

Hi Tony, this is not related to the tests that were added in JRE8. Take a look at what this assembly block is doing in the pom and then take a look for the corresponding warning in the log files. Once you define / assemble files in the pom you need to use the assembled file/artifact in the docker file.